CHANGELOG.md

```diff
@@ -12,8 +12,14 @@ All notable changes to this project will be documented in this file.  The format
 
 ## [Unreleased]
 
+## [2.0.0] - 2023-06-14
+
+
+
 ## [1.1.0] - 2023-06-13
```

Comment:
We tend to mark the yanked version (like [this example](https://github.com/casper-network/casper-node/blob/dev/types/CHANGELOG.md#160-yanked)).

Also, it would be worth adding at least one line to version 2.0.0 to say what changed (like [this example](https://github.com/casper-network/casper-node/blob/dev/types/CHANGELOG.md#fixed-1)).

Reply:
Done in https://github.com/casper-network/casper-wasm-utils/pull/9/commits/dacfa9655500a924b41601fb05ff77a2a8141f9d